src/components/Categories/GetCategory.vue

```diff
@@ -33,10 +33,12 @@
                 <hr />
                 <div class="row" :style="{marginTop: '20px',marginBottom: '13px'}">
                     <div class="col-sm-12">
-                    <input type = "submit" class="btn btn-primary" target="__blank" value="Cập nhật" @click="routeToEdit(category.id)"
-                    :style="{borderRadius: '10px'}" />
-                    <input type = "submit" class="btn default" target="__blank" value="Xóa" @click="deleteCategory()" 
-                    :style="{marginLeft:'10px', border: '1px solid black', borderRadius:'10px'}" />
+                    <Button type = "submit" :hidden="!isInRole()" class="btn btn-primary" target="__blank" @click="routeToEdit(category.id)"
+                    :style="{borderRadius: '10px'}">Cập nhật</Button>
```

Comment:
Mấy cái style này hoàn toàn không cần ấy, check bên thư viện có sẵn rồi

Reply:
done